src/Adapter/MSTestAdapter.PlatformServices/Discovery/AssemblyEnumerator.cs

```diff
@@ -322,12 +322,16 @@ static string GetMethodName(MethodInfo methodInfo)
                 : methodInfo.Name;
         }
 
-        static UnitTestElement GetFixtureTest(string classFullName, string assemblyLocation, string fixtureType, string methodName, string[] hierarchy)
+        static UnitTestElement GetFixtureTest(string classFullName, string assemblyLocation, string fixtureType, string methodName, string[] hierarchy, MethodInfo methodInfo)
         {
-            var method = new TestMethod(classFullName, methodName, hierarchy, methodName, classFullName, assemblyLocation, null, TestIdGenerationStrategy.FullyQualified);
+            string displayName = $"[{fixtureType}] {methodName}";
```

Comment:
where did this previously come from?
can fixtureType be null or empty?

Reply:
It was previously on line 330. I extracted it to a variable to pass it to both UnitTestElement and TestMethod. I'll follow-up to refactor this and have display name only in a single place.